pandas/tests/extension/json/array.py

```diff
@@ -27,7 +27,7 @@
 class JSONDtype(ExtensionDtype):
     type = abc.Mapping
     name = "json"
-    na_value = UserDict()
+    na_value = UserDict()  # type: UserDict
```

Comment:
can you type this as `Mapping[<type>, <type>]`

Reply:
So since this test is for `JSONDtype` we are looking at a `Mapping[str, Any]` correct? A custom JSON Type was/is discussed in python/typing#182

Reply:
I think `UserDict[str, Any]` works

Reply:
this will produce `TypeError: 'ABCMeta' object is not subscriptable` using python 3.6 syntax and will require a workaround, see https://mypy.readthedocs.io/en/latest/common_issues.html#using-classes-that-are-generic-in-stubs-but-not-at-runtime so probably best to use Mapping from typing.

Reply:
> I think `UserDict[str, Any]` works

This is OK. probably better than Mapping. just needs to be quoted for py3.6